charts/drupal/templates/_helpers.tpl

```diff
@@ -373,23 +373,21 @@ done
 {{- define "drupal.extract-reference-data" -}}
 set -e
 if [[ "$(drush status --fields=bootstrap)" = *'Successful'* ]] ; then
-  # Figure out which tables to skip.
-  IGNORE_TABLES=""
-  IGNORED_TABLES=""
-  for TABLE in `drush sql-query "show tables;" | grep -E '{{ .Values.referenceData.ignoreTableContent }}'` ;
+  echo "Dump reference database."
+  mkdir /tmp/reference-data/
```

Comment:
any change this dir already exists?
if yes, then the contents may become mixed, if table is removed from db.
if yes, then mkdir fails, because the dir already exists, or it needs to have `-p`.
if no, then all good.

Reply:
Hmm, to be on the safe side I renamed the directory to /tmp/reference-data-export/. There is no chance that such directory already exists :)